apps/sensenet/src/components/drawer/PermanentDrawer.tsx

```diff
@@ -127,54 +129,98 @@ export const PermanentDrawer = () => {
-                    aria-label={item.primaryText}
-                    className={classes.listButton}
-                    button={true}
+                item.primaryText !== 'Settings' && (
+                  <NavLink
+                    aria-label={item.url}
+                    to={item.url}
+                    className={classes.navLink}
                     key={index}
-                    selected={!!matchPath(location.pathname, item.url)}
-                    data-test={`drawer-menu-item-${item.primaryText.replace(/\s+/g, '-').toLowerCase()}`}>
-                    <ListItemIcon
-                      className={clsx(classes.listItemIconDark, globalClasses.centered, {
-                        [classes.listItemIconLight]: personalSettings.theme === 'light',
-                      })}>
-                      <Tooltip title={item.secondaryText} placement="right">
-                        <div>{item.icon}</div>
-                      </Tooltip>
-                    </ListItemIcon>
-                    {opened && (
+                    onClick={() => {
+                      selectionService.activeContent.setValue(undefined)
+                    }}
+                    activeClassName={classes.navLinkActive}>
+                    <ListItem
+                      aria-label={item.primaryText}
+                      className={classes.listButton}
+                      button={true}
+                      key={index}
+                      selected={!!matchPath(location.pathname, item.url)}
+                      data-test={`drawer-menu-item-${item.primaryText.replace(/\s+/g, '-').toLowerCase()}`}>
+                      <ListItemIcon
+                        className={clsx(classes.listItemIconDark, globalClasses.centered, {
+                          [classes.listItemIconLight]: personalSettings.theme === 'light',
+                        })}>
+                        <Tooltip title={item.primaryText} placement="right">
+                          <div>{item.icon}</div>
+                        </Tooltip>
+                      </ListItemIcon>
+                      {opened && (
+                        <Tooltip title={item.secondaryText} placement="right">
+                          <ListItemText
+                            primary={`${item.primaryText}`}
+                            style={{
+                              color:
+                                theme.palette.type === 'light'
+                                  ? theme.palette.common.black
+                                  : theme.palette.common.white,
+                            }}
+                          />
+                        </Tooltip>
+                      )}
+                    </ListItem>
+                  </NavLink>
+                )
+              )
+            })}
+          </li>
+          <li>
+            {settingsItem && (
+              <NavLink
+                aria-label={settingsItem.url}
+                to={`${settingsItem.url}stats`}
```

Comment:
not this way.
`to={settingsItem.url` here
and the url should be corrected on `use-drawer-items.tsx` (commented there)